Translation revisions (#561)
07th-mod/higurashi-console-arcs: Update/koto_006.txt

//rいっぱいに伸びた枝はたわわにしだれて下がり、その隙間から零れ落ちるように、輝く太陽が光の粒となってきらきらとざわめいている。
	OutputLine(NULL, "いっぱいに伸びた枝はたわわにしだれて下がり、その隙間から零れ落ちるように、輝く太陽が光の粒となってきらきらとざわめいている。",

> 			NULL, "The dazzling sun became particles of light like sparkling raindrops as it shone between the long, drooping branches of the trees.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The dazzling sun became particles of light, like sparkling raindrops, as it shone between the long drooping branches of the trees.", Line_Normal);
```

			NULL, "\"Come to think about it, you told me about \"assimilation\" once...", Line_WaitForInput);
	PlaySE(4, "ps3/s16/46/ds43020490", 256, 64);
	OutputLine(NULL, "…ひょっとして私の中にも、あなた方の同朋の魂魄(こんぱく)が眠っているのでしょうか？」",

> 			NULL, " Could I possibly have the spirits of one of your brethren dwelling within me?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Could I possibly have the spirit of one of your brethren dwelling within me?\"", Line_Normal);
```

//rおそらく、…移住当初は想像を超える苦難があったと思う。
	OutputLine(NULL, "おそらく、…移住当初は想像を超える苦難があったと思う。",

> 			NULL, "Things were likely unimaginably harsh for the initial immigrants.", Line_Normal);

Reviewer: ```suggestion
			NULL, "In all likelihood... things were unimaginably harsh for the initial immigrants.", Line_Normal);
```
